common/src/main/scala/hmda/serialization/filing/FilingCommandsSerializer.scala

```diff
@@ -49,6 +50,8 @@ class FilingCommandsSerializer(system: ExtendedActorSystem)
       updateSubmissionToProtobuf(cmd, resolver).toByteArray
     case cmd: GetLatestSubmission =>
       getLatestSubmissionToProtobuf(cmd, resolver).toByteArray
+    case cmd: GetSubmissionSummary =>
+      getSubmissionSummaryToProtobuf(cmd, resolver).toByteArray
```

Comment:
I think you need the other conversion as well in this file

Reply:
Great catch! Updated.